File: Dice-Python-v2/game.py
import set

class Game:
    setsLimit = 0
    def __init__(self, setsLimit):
        self.setsLimit = setsLimit
        self.results = []
    
    def play(self):
        for i in range(0,self.setsLimit):
            s = set.Set(100)
            self.results.append(s.play())
        return self.results

File: Dice-C-sharp-v5/Game.cs
namespace Puzzle;
public class Game
{
    // public int SetsLimit { get; set; } = 1000;
    public List<int> Play(int limit, int stopPoint)
    {
        List<int> results = new List<int>();
        for (int i = 0; i < limit; i++)
        {
            var set = new Set(stopPoint);
            results.Add(set.Play());
        }

        return results;

    }

}

